<ui version="4.0" >
 <class>LatexEditor</class>
 <widget class="QDialog" name="LatexEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Enter Code:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="sourceTextEdit" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="updatePushButton" >
           <property name="text" >
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="revertPushButton" >
           <property name="text" >
            <string>Revert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>273</width>
         <height>315</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Options</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Resolution:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSpinBox" name="dpiSpinBox" >
            <property name="specialValueText" >
             <string>Automatic</string>
            </property>
            <property name="suffix" >
             <string> DPI</string>
            </property>
            <property name="maximum" >
             <number>10000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Program:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="programComboBox" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="preambleCheckBox" >
          <property name="text" >
           <string>Use Preamble</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Program Messages:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="messagesTextEdit" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="undoRedoEnabled" >
           <bool>false</bool>
          </property>
          <property name="lineWrapMode" >
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QPushButton" name="externalEditorPushButton" >
            <property name="text" >
             <string>Run external editor...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="killPushButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Kill Program</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="statusLabel" >
       <property name="text" >
        <string>Status: Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceTextEdit</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>updatePushButton</tabstop>
  <tabstop>revertPushButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>dpiSpinBox</tabstop>
  <tabstop>programComboBox</tabstop>
  <tabstop>preambleCheckBox</tabstop>
  <tabstop>messagesTextEdit</tabstop>
  <tabstop>externalEditorPushButton</tabstop>
  <tabstop>killPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
